pkg/host/host_openbsd.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package host
+
+import (
+	"github.com/google/syzkaller/prog"
+)
+
+func isSupported(c *prog.Syscall, sandbox string) (bool, string) {
+	return true, ""
+}
+
+func EnableFaultInjection() error {
+	return nil
+}
+
+func init() {
+	checkFeature[FeatureCoverage] = unconditionallyEnabled
```

Comment:
This looks wrong, because we don't have coverage support in executor. I guess you copied it from netbsd, but I got it wrong after a refactoring. Now fixed in:
https://github.com/google/syzkaller/commit/bf3cf0c965acde4217280429327c9012b37aaf17

So just drop this whole init function.